deployJenkinsfile

```diff
@@ -7,6 +7,41 @@ pipeline {
         }
     }
     stages {
+        stage('run migration') {
+            agent {
+                dockerfile {
+                    filename 'migrationDockerfile'
+                    label 'team:iow'
+                }
+            }
+            steps {
+                script {
+                    def deployStage = "$ENVIRONMENT".toUpperCase()
+                    def scriptCmd = "/usr/local/bin/aws ssm get-parameter --name \"/aws/reference/secretsmanager/MON-LOC-$deployStage\" --query \"Parameter.Value\" --with-decryption --output text --region \"us-west-2\""
+                    def secretsString = sh(script: scriptCmd, returnStdout: true).trim()
+                    def secretsJson = readJSON text: secretsString
+                    env.DATABASE_HOST = secretsJson.DATABASE_ADDRESS
```

Comment:
You might want to point out to Ed which environment variables should be removed once you two both merge. So whoever merges last I guess.

Reply:
I've remove a bunch of them.